tests/test_license.py

```diff
@@ -1,58 +1,75 @@
-#!/usr/bin/env python3
-
-# SPDX-FileCopyrightText: 2022 Henrik Sandklef
+# SPDX-FileCopyrightText: 2022 we, the people
```

Comment:
Nope - either add your own CR as an additional line or keep it as it is

Reply:
But it was funny, though.

Reply:
that was the idea ;)